Dell/recovery_backend.py

```diff
@@ -777,6 +777,31 @@ def validate_driver_package(self, package, sender=None, conn=None):
         logging.debug("Validation complete: valid %s" % valid)
         self.report_package_info(valid, description, error_warning)
 
+    @dbus.service.method(DBUS_INTERFACE_NAME,
+        in_signature = 'b', out_signature = '', sender_keyword = 'sender',
+        connection_keyword = 'conn')
+    #The funcution is used to recovery DHC
+    def enable_boot_to_restore_dhc(self, reboot, sender=None, conn=None):
+        """Enables the default one-time boot option to be recovery"""
+        self._reset_timeout()
+        recovery=find_partition().decode('utf-8')
+        mntdir=self.request_mount(recovery, 'rw', sender,conn)
+        #Create dhc_flag to provide judgment for recovery DHC to delete copied dhc file, and dhc_flag will be deleted after recovery DHC.  
+        os.makedirs(os.path.join(mntdir, "dhc_flag"))
```

Comment:
I don't see this used anywhere else in the PR  What's it used by?

Reply:
This used to switch recovery Ubuntu and recovery DHC.  After recovery DHC, the scripts and seed files we copied about DHC  are still kept in the corresponding directory. If  next recovery Ubuntu at this time, then the flow of recovery DHC is still the same.  So create a script in DHC/scripts/chroot-scripts/fish  to Check if there is this flag, and if have, delete the file copy during recovery DHC.

Reply:
Just as a point of preference, would you make it `.dhc_flag` instead of `dhc_flag`?  Then it's hidden by default in the filesystem when you use something like `ls`.